app/javascript/components/CreateLogin.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from "react";
+import { useDispatch } from "react-redux";
+import createUser from "../Redux/UserAction";
+import { useNavigate } from "react-router-dom";
+import { Form } from "react-bootstrap";
+import { Button } from "react-bootstrap";
+import NavBar from "./Navbar";
+
+const CreateUser = () => {
+
+  const [user, setUser] = useState()
+
+  const dispatch = useDispatch()
+
+  const navigate = useNavigate();
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    try {
+      const response = await fetch("/api/v1/users", {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify({ username: user }),
+      });
+      const result = await response.json();
+      console.log(result.data);
```

Comment:
- [ ] Please, kindly remove this old piece of debugging code to improve the cleanliness of the repo 👍